### ISSUE TITLE
Consider adding more commentary checkboxes 

### ISSUE THREAD

**Issue by nonamethanks** [danbooru/danbooru#4351]
It'd be really useful to include a checkbox for "english commentary", which is the second most common type of commentary other than japanese.

Currently the commentary window looks like this:
![image](https://user-images.githubusercontent.com/12946050/77238903-1bfd1b80-6bd5-11ea-8022-6c6d4c9e95b6.png)

However these only cover the case in which a commentary is in japanese and is being translated.
Over the past three years multiple tags for commentaries other than japanese have been introduced, which has rendered these checkboxes obsolete.

Four more checkboxes can be inserted on the right without it becoming too cluttered, since the space's already being taken but basically wasted, because the checkboxes are one per line.
Adding right-aligned checkboxes would be basically free.

From the Discord:

> [00:30] Laethiel: Maybe add checkboxes for English, Korean, Chinese, and symbol commentary on the right?

**Comment by BrokenEagle** [comment]
No... let's not add a bunch of additional clutter to that commentary dialog.

As far as being obsolete, well that's not true. I use most of those checkboxes all of the time. For those that are actually translating commentary, which is what that dialog box is for, those are the only inputs that should be needed. All of those other items you mentioned should be added by the uploader, and if not, then the one adding them might as well add other tags while they're at it since the uploader apparently did a poor job on that upload.

As far as wasted space goes, well that's easily solved with CSS and/or HTML restructuring. I'm already doing so with my own custom CSS.

![image](https://user-images.githubusercontent.com/21149935/77242301-4db4b900-6bba-11ea-8c31-e4adc9a35d2f.png)


```css
.new_artist_commentary .input.boolean.optional,
.edit_artist_commentary .input.boolean.optional {
    display: inline-block !important;
    margin: 2px !important;
    width: 260px !important;
}
```

I would much rather see that being formalized than the other thing.

**Comment by nonamethanks** [comment]
I'll consider this fixed by pull #5950.
